Makefile

```
@@ -1,6 +1,6 @@
 SHELL := /bin/bash
 
-.PHONY: deploy deploy-auto invalidate destroy test test-unit test-terraform test-all coverage security
+.PHONY: deploy deploy-auto invalidate destroy test test-unit test-terraform test-all coverage security lint
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)